src/amo/components/CollectionManager/index.js

```diff
@@ -156,23 +179,22 @@ export class CollectionManagerBase extends React.Component<Props, State> {
 
   render() {
     const {
-      collection, errorHandler, i18n, isCollectionBeingModified, siteLang,
+      collection,
+      creating,
+      currentUsername,
+      errorHandler,
+      i18n,
+      isCollectionBeingModified,
+      siteLang,
     } = this.props;
     const { name, slug } = this.state;
 
-    let collectionUrlPrefix = '';
-    if (collection && siteLang) {
-      const apiHost = config.get('apiHost');
-      const { authorUsername } = collection;
-      collectionUrlPrefix =
-        `${apiHost}/${siteLang}/firefox/collections/${authorUsername}/`;
-    }
+    const collectionUrlPrefix =
+      oneLineTrim`${config.get('apiHost')}/${siteLang || ''}/firefox/collections/
```

Comment:
When would `siteLang` ever not be set?

Reply:
I don't see a test for this either, seems like `siteLang` should just always be set.

Reply:
I had to do this for Flow at one point, but with the current code yeah, it's not needed - removing!